frontend/src/views/AllQueues.js

```diff
@@ -52,9 +52,17 @@ const AllQueues = observer(() => {
     api
       .getPrograms()
       .then(({ data }) => {
```

Comment:
we should run the filter before we make the requests to the queue endpoint. That endpoint is relatively expensive, and nothing will come back anyway.

try using Array.filter method, itll be a little more clear what is going on

Reply:
So do you think where I'm implementing the filter is ok @MikeyManoguerra ?

Reply:
Yes, If its something that a more common operation we could seperate it out into a fn, but this is the only place we are doing it so far